assets/app/views/project.html

```diff
@@ -16,8 +16,14 @@
         <div class="row">
           <div class="col-md-12">
             <div class="tile">
-              <h2 class="service">{{serviceId}}
-                <span class="small"> - routing {{service.spec.protocol}} traffic on {{service.spec.portalIP}}:{{service.spec.port}} to port {{service.spec.containerPort}}</span>
+              <h2 class="service" ng-init="numPorts = service.spec.ports.length">{{serviceId}}
+                <span class="small"> - routing traffic on {{service.spec.portalIP}} -</span>
+                <span class="small" ng-repeat="item in service.spec.ports | orderBy:port | limitTo:2">
+                  port {{item.port}} &#8594; {{item.targetPort}} ({{item.protocol}}){{$index + 1 == numPorts ? '' : ', '}}
```

Comment:
Instead of having the ternary operator embedded like this: `{{$index + 1 == numPorts ? '' : ', '}}`
write it as a span with an ng-if like `<span ng-if="$index + 1 != numPorts">,</span>`